02-Homework/script.js

```diff
@@ -0,0 +1,56 @@
+// Assignment Code
+var generateBtn = document.querySelector('#generate');
+
+function generatePassword() {
+  // PLEASE ADD ALL YOUR CODE HERE
+}
+alert ("text");
+var text
+var intiger
+if ( text = intiger) {
+"check is within range"
+} else {
+  alert and break
+}
+
+var confirm; (lowercase)
```

Comment:
I'm not entirely sure what you are trying to do here, but in any case the syntax is incorrect. the `;` defines the end of a line, so we should never see anything following it.